src/App.tsx

```diff
@@ -19,13 +33,18 @@ export const App: React.FC = () => {
                 type="text"
                 id="search-query"
                 className="input"
+                defaultValue={query}
+                // value={filter}
```

Comment:
remove such things, you don't need them messing up your code

Reply:
regarding that failing test – have you tried putting `query` inside `value` prop? why are you using `defaultValue` prop?